deployment/state/state.go

```diff
@@ -48,6 +49,7 @@ type CCIPChainState struct {
 	FeeQuoter        address.Address
 	Timelock         address.Address
 	ReceiverAddress  address.Address
+	MCMS             address.Address
```

Comment:
nit: Let's group // MCMS contracts separately in these enumeration of contracts - it's a separate decoupled protocol, should not be grouped under misc/utils